object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/ObjectConstructionTransfer.java

```diff
@@ -74,9 +88,54 @@ public ObjectConstructionTransfer(final CFAnalysis analysis) {
             result.getElseStore(),
             exceptionalStores);
     exceptionalStores = null;
+
+    Node receiver = node.getTarget().getReceiver();
+    addTemporaryVars(result, node);
```

Comment:
I'd move this line before the previous one, to keep the code about the `receiver` together.  Then, add some comments indicating what the code involving `receiver` is doing